datasets/noaa-mrms-qpe/tests/test_dataset.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from pathlib import Path
+
+from pctasks.cli.cli import setup_logging, setup_logging_for_module
+from pctasks.dev.test_utils import run_process_items_workflow
+
+HERE = Path(__file__).parent
+DATASET_PATH = HERE / "dataset-dev-cluster.yaml"
+
+
+def test_1h_pass1():
+    run_process_items_workflow(
+        DATASET_PATH,
+        "noaa-mrms-qpe-1h-pass1",
+        args={
+            "registry": "localhost:5001",
+        },
+        splits_limit=1,
+        chunks_limit=2,
+        timeout_seconds=600,
+    )
```

Comment:
We have pytest in pctasks right? You could parametrize this to deduplicate the test code.

```suggestion
@pytest.mark.parametrize("collection", ["noaa-mrms-qpe-1h-pass1", "noaa-mrms-qpe-1h-pass2", "noaa-mrms-qpe-24h-pass2"])
def test_1h_pass1(collection):
    run_process_items_workflow(
        DATASET_PATH,
        collection,
        args={
            "registry": "localhost:5001",
        },
        splits_limit=1,
        chunks_limit=2,
        timeout_seconds=600,
    )
```

Reply:
Done.